### DISCUSSION TITLE
Osu Lazer Still Not Submitting Scores On Linux

### DISCUSSION THREAD

**Discussion by SltySptoon** [ppy/osu#27246] [Q&A]
In the image it says I'm running the most recent version of osu but when I play a map it still gives me the error message saying it won't submit my score. My logs are also linked below
[compressed-logs.zip](https://github.com/ppy/osu/files/14335260/compressed-logs.zip)
![unnamed](https://github.com/ppy/osu/assets/60854876/17ec9860-883a-4312-9311-d74e4b79e5d7)

**Comment by bdach** [comment]
how are you even getting the game to run on steam deck?

**Reply by SltySptoon** [reply]
I just downloaded osu from the official download page and ran the app image

**Reply by bdach** [reply]
then it must be something about steam deck that is preventing the measures we have in place to make sure everyone is playing fair from working. dunno what it is, we don't officially support the platform.
